packages/extension-sample/src/setup-app.ts

```diff
@@ -21,6 +21,10 @@ class Sample implements IApplicationExtension {
         this.config = config
     }
     
+    getConfig(): object {
```

Comment:
To use the same type as what's defined in the `IApplicationExtension` interface:
```suggestion
    getConfig(): Record<string, any> {
```